boltdesignsystem/bolt
# DS-291 Popover custom width and docs rewrite

## Jira

https://pegadigitalit.atlassian.net/browse/DS-291

## Summary

Adds a new CSS var for customizing a popover's bubble width.

## Details

1. Component specific CSS var is created for the bubble width.
2. All docs are rewritten with the new template.
3. `boundary` is documented along with `placement`.
4. Updated no-js CSS so that the popover has no width or height when it's not activated.

## How to test

Run the branch locally and go through all the popover docs. Make sure everything works as expected.
